test/functional/cmdLineTests/classesdbgddrext/playlist.xml

```diff
@@ -51,6 +51,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<variations>
 			<variation>Mode110</variation>
 			<variation>Mode610</variation>
+			<variation>Mode500</variation>
```

Comment:
The ideal is to find the equivalent balanced mode.

Mode500 is `-Xint -Xgcpolicy:balanced -Xnocompressedrefs`. We should not use `-Xint` unless the test is specifically testing `-Xint`.

In this case, we should set Mode501 and Mode551.

https://github.com/adoptium/TKG/blob/c0fd937c76392ba409c11608c1308e6d1c89b8af/resources/modes.xml#L3188
https://github.com/adoptium/TKG/blob/c0fd937c76392ba409c11608c1308e6d1c89b8af/resources/modes.xml#L3348